addons/l10n_uy/models/template_uy.py

```diff
@@ -33,3 +35,29 @@ def _get_uy_res_company(self):
                 'account_purchase_tax_id': 'vat4',
             },
         }
+
+    @template('uy', 'account.journal')
+    def _get_uy_account_journal(self):
+        return {
+            'sale': {
+                "name": _("Customer Invoices"),
+                "code": "0001",
+                "l10n_latam_use_documents": True,
+                "refund_sequence": False,
+            },
+            'purchase': {
+                "name": _("Purchases"),
+                "code": "0002",
+                "l10n_latam_use_documents": True,
+                "refund_sequence": False,
+            },
+        }
+
+    def _load(self, template_code, company, install_demo):
+        """ Set companies country by default after install the chart of account,
+        also set the rut as the company identification type because this one is the uruguayan vat """
+        res = super()._load(template_code, company, install_demo)
+        if template_code == 'uy':
```

Comment:
We do it for AR loc but not sure is there is a generic code that does it?

https://github.com/odoo/odoo/blob/8e6a3d7269f81191ec8180e6d5ee8d403238f9a2/addons/l10n_ar/models/account_chart_template.py#L21-L45

1. I think that the identification type of the partner-company should be set to default marked as is_vat. There is a generic code that does it? I notice this is made for creating new companies on the l10n_latam_base module but not to existing companies when installing CoA. 

2. related to setting the country on the company. Also think is a good thing to do. You think this could be a problem?

@ren-odoo @fvz-odoo please give us your feedback

Reply:
@jco-odoo in the Locs where we don't have correct  the identification type we need to manually update the value when doing the testing, so if we can set as default in the demo is better from the Funtional point of view. 

cc. @fvz-odoo @zaoral

Reply:
We might do this in generic as well.  There is a default, which is the generic VAT but we could make it a compute.  For AR,there are multiple CoA, which makes it more logical.  For sure, we should not set the country.  We could keep the identification type, but VAT vs RUT, it should not have much difference in behaviour.

Reply:
@jco-odoo 
Ok. I will remov the country and leave the indentitication type by the moment